### ISSUE TITLE
your tiny tool is too big :)

### ISSUE THREAD

**Issue by tothpaul** [taviso/hotcorner#20]
I've tried the same thing with Delphi...and it's smaller :p

https://github.com/tothpaul/Delphi-LowLevel/tree/master/hotcorner

**Comment by taviso** [comment]
Haha, sounds like a challenge! 😃

**Comment by tothpaul** [comment]
Yep ;) but wait, I've forgot to add two compiler switches to remove RTTI and strip the rellocation table, now it's only 40 448 bytes :D and it still use the standard system unit ;)
